sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.1.0-preview.6 (Unreleased)
+## 1.1.0 (Unreleased)
```

Comment:
@jonathandturner I made this change as an attempt to fix CI.

Reply:
I wonder, what else should we put in this CHANGELOG entry? It could be done in another PR, but I'm interested in how you would approach this (as I'm working on other changelog entries too, I think I can learn from your approach).